kong/plugins/acme/handler.lua

```diff
@@ -59,6 +59,16 @@ function ACMEHandler:init_worker()
   ngx.timer.every(86400, client.renew_certificate)
 end
 
+local function verify_domains(conf, host)
```

Comment:
```suggestion
local function check_domain(conf, host)
```

Reply:
since we are not doing verification, and to avoid confusion with the validation that ACME did itself, let's use another word